ui/lib/passport-expiration.ts

```diff
@@ -0,0 +1,24 @@
+import { ethers } from "ethers";
+import { useMemo } from "react";
+import { useAccount } from "./use-wagmi";
+import { useVeNationLock } from "./ve-token";
+
+const decimalThreshold = Number(process.env.NEXT_PUBLIC_CITIZENSHIP_REVOKE_THRESHOLD)
+const threshold = ethers.BigNumber.from(String(decimalThreshold * 10 ** 18));
+const maxLockPeriod = ethers.BigNumber.from(31556926 * 4);
```

Comment:
Can you make the number `31556926` human-readable?  Or add a comment explaining what it is?